Client/Frontend/Browser/Onboarding/OnboardingRewardsAgreementViewController.swift

```diff
@@ -33,22 +44,23 @@ class OnboardingRewardsAgreementViewController: OnboardingViewController {
     }
     
     override func continueTapped() {
+        rewards?.ads.isEnabled = true
```

Comment:
Correct. Honestly I purposely changed the name to "continue" because this one button has had it's fair share of changes lol.. "ShowMe", "Join", "Turn On", "Continue", etc.. I just decided "continue" will be generic enough that if I change the title it just means it continues to the next screen.

Reply:
The main issue is you can't enable rewards like this anymore. It will be enabled automatically based on if your region is supported when you create the wallet. The only time you have to set this to true is when you're turning on ads for existing users who have enabled rewards already when their region was unsupported and now their region is supported

Reply:
@Brandon-T ☝️

Reply:
Fixed.